{
  "d91e7b4392163e0b46289a9f8b97b16c" : {
    "statistic" : {
      "failed" : 0,
      "broken" : 1,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "c8d28dc6dc792d6d",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.TimeoutException: Message:",
      "time" : {
        "start" : 1600420574901,
        "stop" : 1600420617349,
        "duration" : 42448
      }
    } ]
  },
  "fa363551da3b313dda3c470cf52ad218" : {
    "statistic" : {
      "failed" : 0,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 1,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "5ca256933ec67e7",
      "status" : "passed",
      "time" : {
        "start" : 1600656333365,
        "stop" : 1600656352373,
        "duration" : 19008
      }
    } ]
  },
  "12451970f50aab0ec4d353f609012c2f" : {
    "statistic" : {
      "failed" : 0,
      "broken" : 1,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "d65386758407ff0e",
      "status" : "broken",
      "statusDetails" : "requests.exceptions.ConnectionError: HTTPConnectionPool(host='calapi.51jirili.comhttp', port=80): Max retries exceeded with url: //calapi.51jirili.com/config/common (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000027B71720B50>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))",
      "time" : {
        "start" : 1600414476516,
        "stop" : 1600414476566,
        "duration" : 50
      }
    } ]
  },
  "dd0646a5c1a798e47c4f25747509dea9" : {
    "statistic" : {
      "failed" : 0,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 1,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "73bd88f681fbaa2f",
      "status" : "passed",
      "time" : {
        "start" : 1600656212877,
        "stop" : 1600656230279,
        "duration" : 17402
      }
    } ]
  },
  "673c6cf796d54f5cbc2e9852ef4f69c7" : {
    "statistic" : {
      "failed" : 0,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 1,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "80843a1c3c713529",
      "status" : "passed",
      "time" : {
        "start" : 1600656271092,
        "stop" : 1600656333352,
        "duration" : 62260
      }
    } ]
  },
  "28dda5313ff182aeade3becf47df4372" : {
    "statistic" : {
      "failed" : 0,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 1,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "6a5b384b61244eda",
      "status" : "passed",
      "time" : {
        "start" : 1600656230304,
        "stop" : 1600656271081,
        "duration" : 40777
      }
    } ]
  },
  "d85ebd52118862a52004a5c4faaac530" : {
    "statistic" : {
      "failed" : 0,
      "broken" : 1,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "701a8bffd7f44ef3",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"body > div:nth-child(6) > div > div.ant-modal-wrap > div > div.ant-modal-content > div.ant-modal-body > form > div:nth-child(1) > div > div > div.ant-col.ant-col-14.ant-form-item-control-wrapper > div > span > input\"}\n  (Session info: chrome=74.0.3729.108)\n  (Driver info: chromedriver=74.0.3729.6 (255758eccf3d244491b8a1317aa76e1ce10d57e9-refs/branch-heads/3729@{#29}),platform=Windows NT 10.0.18362 x86_64)",
      "time" : {
        "start" : 1600656352400,
        "stop" : 1600656367611,
        "duration" : 15211
      }
    } ]
  },
  "a55d1468f36f1b2cdddeddca0f290d7d" : {
    "statistic" : {
      "failed" : 0,
      "broken" : 1,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "aadff1b073f70377",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: chrome not reachable\n  (Session info: chrome=74.0.3729.108)\n  (Driver info: chromedriver=74.0.3729.6 (255758eccf3d244491b8a1317aa76e1ce10d57e9-refs/branch-heads/3729@{#29}),platform=Windows NT 10.0.18362 x86_64)",
      "time" : {
        "start" : 1600656367797,
        "stop" : 1600656407615,
        "duration" : 39818
      }
    } ]
  }
}